release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3318,6 +3318,63 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, if egress IPs were hosted by any interface other than the primary interface, there was no verification to determine if a next hop is required. With this release, the IP will inspect the main routing table and determine if a next hop is required. (link:https://issues.redhat.com/browse/OCPBUGS-31854[*OCPBUGS-31854*])
+
+[id="ocp-4-14-23-profile-added-drop-unsupported-parameters"]
+===== New profile for RT kernel to drop unsupported parameters
+
+* Previously, the `net.core.busy_read`, `net.core.busy_poll`, and `kernel.numa_balancing sysctl` parameters did not exist within the RT kernel and were therefore unsupported. With this release, the `openshift-node-performance-rt` profile is added and included if an RT kernel is detected, which drops the unsupported kernel parameters before they are applied. (link:https://issues.redhat.com/browse/OCPBUGS-31905[*OCPBUGS-31905*])
+
+[id="ocp-4-14-23-disable-OLM-disconnected"]
+===== Disable option for OLM default source
+
+* Previously, there was no way to disable the Operator Lifecycle Manager (OLM) default source in a disconnected situation. With this release, the `OperatorHubSpec` field is integrated into the `hostedcluster.Spec.Configuration` API to facilitate disabling and enabling default sources during creation. The CLI also includes a flag for this functionality. (link:https://issues.redhat.com/browse/OCPBUGS-32221[*OCPBUGS-32221*])
+
+[id="ocp-4-14-23-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Node Tuning Operator (NTO) checked if there were profiles that share the same priority, regardless of their associated node. The process was for the NTO to first collect the profiles, check for priority conflicts, and then filter for the associated node. As a result, if multiple performance profiles were present on two different nodes, false priority warnings were dumped into the logs. With this release, the steps of this process have been changed so that the NTO filters the associated nodes first and then checks for priority conflicts. (link:https://issues.redhat.com/browse/OCPBUGS-31735[*OCPBUGS-31735*])
+
+* Previously, there was a fundamental issue that prevented egress IPv6 from working for Elastic IPs (EIPs) with multi-network interface controllers (NICs). With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31853[*OCPBUGS-31853*])
+
+* Previously, the `kube-rbac-proxy-crio` label had mistyped resources. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32260[*OCPBUGS-32260*])
+
+* Previously, `v4-transit-switch-subnet` and `v6-transit-switch-subnet` flags were added, but some configurations were still missing to allow setting custom subnets while creating a kind cluster. With this release, it is possible to override default v4 and v6 transit switch subnets while creating a kind cluster. (link:https://issues.redhat.com/browse/OCPBUGS-32353[*OCPBUGS-32353*])
```

Comment:
> some configurations were still missing to allow setting custom subnets 
This seems to be missing something. Not sure if this is correct. 
```suggestion
* Previously, `v4-transit-switch-subnet` and `v6-transit-switch-subnet` flags were added, but some configurations that allow for setting custom subnets while creating a kind cluster were still missing. With this release, it is possible to override default v4 and v6 transit switch subnets while creating a kind cluster. (link:https://issues.redhat.com/browse/OCPBUGS-32353[*OCPBUGS-32353*])
```

Reply:
Does a typical user know what a _kind cluster_ is?

Reply:
Good question - confirming with dev

Reply:
Removing this bug - this feature has two parts, ovn-kubernetes side of things and cluster network operator side of things. Dev has finished backporting ovn-kubernetes part to 4.14 but cluster-b=network-operator side of things have not yet made it to 4.14.